src/components/menu/index.jsx

```diff
@@ -30,10 +30,16 @@ function Item({ item, inDropdown = false }) {
 }
 
 function Dropdown({ title, items }) {
+  // Use isOpen to show menu on hover.
+  const [isOpen, setOpen] = useState(false);
   // React Bootstrap does not support parent items that are links, only text.
   // Therefore, the parent item it only used for its title.
   return (
-    <NavDropdown title={title}>
+    <NavDropdown title={title}
+                 onMouseEnter = {() => setOpen(true)}
+                 onMouseLeave = {() => setOpen(false)}
+                 show={ isOpen }
```

Comment:
Perhaps we could add a fade here? https://react-bootstrap.github.io/utilities/transitions/#fade